### PR TITLE
CNV-module-cleanup

### PR DESCRIPTION
**Pull request by aburdenthehand**
RDP console feature content was added to 2.0 many moons before we had any assemblies. These two modules are now in the 'Accessing virtual machine consoles' assembly.
There is also a duplicate module (vm wizard reference table) that is getting the ol' Tele Bye Byes.

No BZ or JIRA

### CODE REVIEW

File: modules/cnv-vm-rdp-console-web.adoc [hunk 1]
```diff
@@ -1,6 +1,6 @@
 // Module included in the following assemblies:
 //
-// * TBD
+// cnv/cnv_users_guide/cnv-accessing-vm-consoles.adoc
 
 [id="cnv-vm-rdp-console-web_{context}"]
 = Connecting to the RDP console
```
Comment (kalexand-rh) [inline modules/cnv-vm-rdp-console-web.adoc]: *_console.rdp_*  uses the 3.x filename highlighting. It should be enclosed in backticks. There are several instances in this file, so please fix them.

**Review by kalexand-rh — COMMENTED** [review]
One pick and it LGTM.

**Comment by aburdenthehand** [comment]
@kalexand-rh thanks for the catch. Fixed and ready.

**Comment by kalexand-rh** [comment]
Thank you for fixing it, @aburdenthehand! Merging.

**Pull request merged**